images/notary/sign.sh

```diff
@@ -31,6 +31,8 @@ SIGNING_KEY_ID=2C8DD3224EF3573A42BD221FA8E0CA3C1C2ADA2F
 INTERMEDIATE_CI_PUBLIC_KEY=/mnt/keys/signing/intermediate_ci_public_key.gpg
 INTERMEDIATE_CI_PUBLIC_KEY_ID=78F3726939CA1B94893B66E8BC86F6FB94429164
 
+UO_INTERMEDIATE_CI_PUBLIC_KEY=/mnt/keys/signing/uo_intermediate_ci_public_key.gpg
```

Comment:
@kotfic I'm trying to figure out why these paths don't match what the rebuild jobs specify to trust, e.g. [here](https://github.com/spack/spack/blob/develop/share/spack/gitlab/cloud_pipelines/stacks/e4s/spack.yaml#L256).  There we look for the key in `/mnt/key/...`, but here it says `/mnt/keys/...`.  So I'll update my PR to remove the `aws` tag so it `spack gpg trusts` this path, once I'm sure what's going on.

Reply:
Nevermind, the signing runner just mounts the keys in a different [location](https://github.com/spack/spack-infrastructure/blob/main/k8s/runners/signing/release.yaml#L78) than  the protected runner deployments do (e.g. [here](https://github.com/spack/spack-infrastructure/blob/main/k8s/runners/large-x86-prot/release.yaml#L76)).  I'll update the spack PR.

Reply:
@kotfic It seems the `spack-intermediate-ci-signing-key` kube secret mounted by the protected builders (e.g. [here](https://github.com/spack/spack-infrastructure/blob/main/k8s/runners/large-x86-prot/release.yaml#L75)) still needs to be updated to include the new UO intermediate public key.  It appears you already updated the `spack-signing-key-encrypted` kube secret mounted by the package signing runner.  Does this sound right to you?

Reply:
Can I assume the name of the new key in that secret will be `uo_intermediate_ci_public_key.gpg`?

Reply:
> Can I assume the name of the new key in that secret will be `uo_intermediate_ci_public_key.gpg`?

Sorry, nevermind, that's a question for @eugeneswalker.